src/server/saved_objects/service/lib/errors.js

```diff
@@ -46,6 +46,12 @@ const CODE_BAD_REQUEST = 'SavedObjectsClient/badRequest';
 export function decorateBadRequestError(error, reason) {
   return decorate(error, CODE_BAD_REQUEST, 400, reason);
 }
+export function createBadRequestError(reason) {
```

Comment:
We're lacking tests for these, and there's pretty good coverage currently.

Reply:
Yeah, I'll add tests for these.